lib/jets/overrides/rails/migration_checker.rb

```diff
@@ -1,11 +1,15 @@
 module ActiveRecord
   module MigrationChecker
     def prepare_test_db
-      current_config = ::ActiveRecord::Base.connection_config
+      current_config = ::ActiveRecord::Base.connection_db_config
```

Comment:
connection_config is deprecated now we should use `::ActiveRecord::Base.connection_db_config` 

deprecation ref:
https://github.com/rails/rails/blob/main/activerecord/lib/active_record/connection_handling.rb#L291